README.adoc

```diff
@@ -428,3 +428,41 @@ In response you should get ID of created item, using that you can retrieve the w
 ----
 $ ./bin/alm-cli show workitem --id <ID> -H localhost:8080 --pp
 ----
+
+==== OpenCompose usage
+
+Running almighty-core with OpenCompose
+
+ * Download kompose release with OpenCompose support from link:https://github.com/redhat-developer/opencompose/releases/tag/v0.1.1[here]. Extract that downloaded kompose release and put in path(This is a temporary solution, future releases will have packages or some form of better delivery mechanism).
+
+
+ * Start a openshift cluster and make sure you can access it from your `oc` command line.
+
+Create openshift project to deploy application
+
+----
+$ oc login -u username -p password
+$ oc new-project almightycore
+----
+
+NOTE: If above commands fail you have no access to cluster.
+
+
+Get services file:
```

Comment:
How about s/Get services file/Deploy the application using OpenCompose services.file : "